python_ta/transforms/ExprWrapper.py

```diff
@@ -26,13 +27,21 @@ class ExprWrapper:
     node: astroid.NodeNG
     types: Dict[str, str]
 
-    def __init__(self, expr: nodes.Expr, types=None):
-        self.node = expr.value
-        if types is None:
-            types = {}
+    def __init__(self, node: astroid.NodeNG, types={}):
         self.types = types
 
-    def reduce(self, node: astroid.NodeNG = None) -> z3.ExprRef:
+        if isinstance(node, astroid.Expr):
+            self.node = node.value  # take node attribute to be the value of the expression
+        elif isinstance(node, astroid.Assign):
+            self.node = node.value  # take node attribute as expression (right side) of assignment
+        elif isinstance(node, astroid.Arguments):
+            self.node = node  # take node attribute to be the function declaration node itself
+        else:
+            raise Z3ParseException(
```

Comment:
raise the more general `ValueError` type; this method isn't doing any "z3 parsing".